Launcher/Classes/ViewControllers/SettingsViewController.swift

```diff
@@ -60,6 +60,11 @@ class SettingsViewController: NSViewController {
         self.dismiss(true)
     }
     
+    @IBAction func clickResetToDefaults(_ sender: Any) {
+        plistOperations.removePlist()
+        AppHandler().restartApplication()
```

Comment:
It would be cool to get the app handler out of this method but maybe it’s hard because it’s the IBAction

Reply:
Not sure that I got this one. We have AppHandler with some actions. Why not reusing these methods in controllers when we need them?

Reply:
I’d like us better pass in an object where we can call methods on or having a property on SettingsViewController. I think its the implementation of the AppHandler that we currently have to call class methods. It would be nicer if we’d pass around an instance but it’s unrelated to this pr, I guess.